### PR TITLE
feat: blocks feed using web-relay

### PR DESCRIPTION
**Pull request by Nuhvi**
Next version using web-relay

### CODE REVIEW

**Comment by Nuhvi** [comment]
@reza don't have permission to tag reviewers

File: lib/writer.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  constructor (client, config, opts) {
+    super(client, config, opts)
+
+    this._config = config
+
+    // Time since start
+    this._age = -1
+
+    // Start on next tick, allowing mocking some methods in unit test after construction.
+    setTimeout(() => {
```
Comment (rbndg) [inline lib/writer.js:22]: Does it make sense to move this to Feeds class?

**Comment by rbndg** [comment]
Other than the question above its look ok

File: lib/writer.js (hunk 1 above)
Comment (Nuhvi) [reply lib/writer.js:22]: This is not really a feature, it helps me mock some methods before the immediate call for `this._onInterval()`. In general implicit logic for intervals is tricky and can throw people off, and hard to generalize for all cases. So let's keep it manual.

**Comment by pwltr** [comment]
This feed is not working for me, doesn't seem to resolve. No errors or anything.

**Comment by Nuhvi** [comment]
@pwltr you have to run a relay and the feed yourself, there is no currentlly deployed relay or version of this feed.

**Comment by pwltr** [comment]
I'm running the relay and the feed, I've got the facts and the news feed working in Bitkit, this is one and the price feed seem to not resolve.

File: lib/slashfeed.json [hunk 2]
```diff
@@ -14,6 +14,12 @@
     "fields": [
         {
             "name": "Block",
+            "description": "Shows the current block information.",
+            "main": "/feed/latest-block",
```
Comment (Nuhvi) [inline lib/slashfeed.json:18]: @rbndg We discussed whether or not to keep an aggregated field `Block` to save network roundtrips for users who want the entire block at once, let me know if I should remove it, at least from `slashfeed.json`.

**Pull request merged**